<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fenetre_rt</class>
 <widget class="QWidget" name="fenetre_rt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RT : Remote Terminal</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>NÂ° de RT voulu :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="choix_rt">
       <property name="minimumSize">
        <size>
         <width>15</width>
         <height>0</height>
        </size>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::PlusMinus</enum>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>30</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bouton_demarrer">
       <property name="text">
        <string>Demarrer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_rt">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>RT</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="indent">
        <number>20</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="listeMessages">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="destinataire">
       <property name="text">
        <string>Destinataire :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="adresse_destinataire">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_1">
       <property name="text">
        <string>Message :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="message"/>
     </item>
     <item>
      <widget class="QPushButton" name="boutonEnvoyer">
       <property name="text">
        <string>Envoyer</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
